src/App.vue

```diff
@@ -731,18 +751,29 @@ export default {
       }
     },
 
-    storeDirectors (response: any): void {
-      const directors = response?.data?.directors
-      if (directors) {
-        const directorsSorted = directors.sort(this.fieldSorter(['lastName', 'firstName', 'middleName']))
-        for (var i = 0; i < directorsSorted.length; i++) {
-          directorsSorted[i].id = i + 1
-          directorsSorted[i].isNew = false
-          directorsSorted[i].isDirectorActive = true
+    storeParties (response: any): void {
+      const parties = response?.data?.parties
+      if (parties) {
+        const directorsList = parties?.filter(partiesArr => partiesArr.roles?.includes(Roles.DIRECTOR))
```

Comment:
FYI, inside the filter call, the variable is single element from the array, in this case just a "party".
No need to change.

Reply:
oh no, thats a good call i will update. 
I was wondering what to call it when trying to differentiate from parties.

Reply:
updated!

Reply:
At some point, probably a good idea to restore the conditional chaining on role type.